### ISSUE TITLE
kenburns-dom standalone usage

### ISSUE THREAD

**Issue by strarsis** [gre/kenburns#4]
When using kenburns-dom standalone, the start method 
deviates from the kenburns module (runStart and run).

Also, runStart of kenburns-dom apparently also doesn't 
return a promise or callback for the finish of the animation.

**Comment by gre** [comment]
I've released a major version of the library (2.0.0) that have a new documentation. https://github.com/gre/kenburns/blob/master/API.md

checkout also the example: https://github.com/gre/kenburns/blob/master/example/index.js


You now only need to use the library `kenburns` and no more `kenburns-dom`.

To use the dom implementation: 

```js
import KenBurnsDOM from "kenburns/lib/DOM";
var div = document.createElement("div");
div.style.width = "400px";
div.style.height = "400px";
const kenBurnsDOM = new KenBurnsDOM(div);
// kenBurnsDOM.animate(...).then(...);
// kenBurnsDOM.animateStep(...);
```

**Comment by gre** [comment]
`animate(image, fromCrop, toCrop, duration/*, easing*/)` runs the animation and returns a Promise.
if you want finer control, `animateStep(image, fromCrop, toCrop, progress)` allows to render one frame of the animation. It's lower level but you can do whatever speed / easing / interruption / ...
